### PR TITLE
[CO-2780] Updated users/identify users/delete document to include email as identifier

### PR DESCRIPTION
**Pull request by shihanxiong**
https://jira.braze.com/browse/CO-2780

Screenshot 2024-07-18 at 7 29 26 PM
Screenshot 2024-07-18 at 7 24 35 PM

### CODE REVIEW

File: _docs/_api/endpoints/user_data/post_user_identify.md [hunk 1]
```diff
@@ -105,22 +105,37 @@ Session data will only be merged if the app exists on both user profiles. For ex
...
 
+### Identifying users by email
+If an `email` is specified as an identifier, an additional `prioritization` value is required in the identifier. The `prioritization` should be an array specifying which user to merge if there are multiple users found. `prioritization` is an ordered array, meaning if more than one user matches from a prioritization, then merging will not occur.
+
+The allowed values for the array are: `identified`, `unidentified`, `most_recently_updated`. `most_recently_updated` refers to prioritizing the most recently updated user.
+
+Only one of the following options may exist in the prioritization array at a time:
+- `identified` refers to prioritizing a user with an `external_id`
+- `unidentified` refers to prioritizing a user without an `external_id`
+
```
Comment (davehensley) [inline _docs/_api/endpoints/user_data/post_user_identify.md:116]: Nit: This whole section is repeated, I wonder if we should DRY it up?

**Review by davehensley — APPROVED** [review]

**Review by brihicks5 — APPROVED** [review]

File: _docs/_api/endpoints/user_data/post_user_identify.md (hunk 1 above)
Comment (shihanxiong) [reply _docs/_api/endpoints/user_data/post_user_identify.md:116]: Yea good q, it's similar to /merge endpoint as well. Not sure if it's possible in doc

File: _docs/_api/endpoints/user_data/post_user_delete.md [hunk 2]
```diff
@@ -48,13 +48,22 @@ Authorization: Bearer YOUR_REST_API_KEY
...
+| -------------- | -------- | -------------------------- | ------------------------------------------------------------------------------------------------ |
+| `external_ids` | Optional | Array of strings           | External identifiers for the users to delete.                                                    |
 | `user_aliases` | Optional | Array of user alias object | [User aliases]({{site.baseurl}}/api/objects_filters/user_alias_object/) for the users to delete. |
-| `braze_ids` | Optional | Array of strings | Braze user identifiers for the users to delete. |
+| `braze_ids`    | Optional | Array of strings           | Braze user identifiers for the users to delete.                                                  |
 {: .reset-td-br-1 .reset-td-br-2 .reset-td-br-3  .reset-td-br-4}
 
+### Deleting users by email
+If an `email` is specified as an identifier, an additional `prioritization` value is required in the identifier. The `prioritization` should be an array specifying which user to merge if there are multiple users found. `prioritization` is an ordered array, meaning if more than one user matches from a prioritization, then merging will not occur.
```
Comment (Vector919) [inline _docs/_api/endpoints/user_data/post_user_delete.md:59]: Should this say `which user to delete` if there are multiple users found?

Comment (Vector919) [inline _docs/_api/endpoints/user_data/post_user_delete.md:59]: There's also a reference to merging later on in this paragraph, not sure if that's intentional

**Review by Vector919 — APPROVED** [review]

File: _docs/_api/endpoints/user_data/post_user_delete.md [hunk 3]
```diff
@@ -48,13 +48,22 @@ Authorization: Bearer YOUR_REST_API_KEY
...
+| `external_ids` | Optional | Array of strings           | External identifiers for the users to delete.                                                    |
 | `user_aliases` | Optional | Array of user alias object | [User aliases]({{site.baseurl}}/api/objects_filters/user_alias_object/) for the users to delete. |
-| `braze_ids` | Optional | Array of strings | Braze user identifiers for the users to delete. |
+| `braze_ids`    | Optional | Array of strings           | Braze user identifiers for the users to delete.                                                  |
 {: .reset-td-br-1 .reset-td-br-2 .reset-td-br-3  .reset-td-br-4}
 
+### Deleting users by email
+If an `email` is specified as an identifier, an additional `prioritization` value is required in the identifier. The `prioritization` should be an array specifying which user to delete if there are multiple users found. `prioritization` is an ordered array, meaning if more than one user matches from a prioritization, then deletion will not occur.
+
+The allowed values for the array are: `identified`, `unidentified`, `most_recently_updated`. `most_recently_updated` refers to prioritizing the most recently updated user.
```
Comment (lydia-xie) [inline _docs/_api/endpoints/user_data/post_user_delete.md:61]:
```suggestion
If an `email` is specified as an identifier, an additional `prioritization` value is required in the identifier. The `prioritization` is an ordered array and should specify which user to delete if multiple users are found. This means deleting users will not occur if more than one user matches a prioritization.

The allowed values for the array are: `identified`, `unidentified`, `most_recently_updated`. `most_recently_updated` refers to prioritizing the most recently updated user.
```

**Pull request merged**